xlml/apis/task.py

```diff
@@ -397,7 +395,7 @@ def run_model(
         )
       launch_workload = self.launch_workload(workload_id, gcs_path)
       wait_for_workload_completion = xpk.wait_for_workload_completion.override(
-          timeout=self.task_test_config.time_out_in_min * 60,
+          timeout=int(self.task_test_config.time_out.total_seconds()),
```

Comment:
I have the tiniest nit: Airflow uses `timeout` instead of `time_out`, so please do the same with out field.

Reply:
Do you mean use `timeout` instead of `time_out`?

Reply:
Yeah, just to keep our naming consistent